scripts/user-actions/display.js

```diff
@@ -3,14 +3,13 @@ console.log('--- loading: display-all.js');
 
 // reverse engineering!
 //   the handler has no tests, study the demo to write this function
-const displayHandler = (message = '', arrayToDisplay = words) => {
+ const displayHandler = (message = '', arrayToDisplay = words) => {
   debugger;
   const combineStrings = (acc, next) => {
-    return _;
+    return acc+'\n'+next;
```

Comment:
I think this is wrong. First one is from Demo and second screenshot from your code...
![Demo Screenshot](https://user-images.githubusercontent.com/62182810/87718872-33cb6180-c7bb-11ea-9200-32e693f73907.PNG)

![Ali screenshot](https://user-images.githubusercontent.com/62182810/87718789-14cccf80-c7bb-11ea-9aac-b3f172959bce.PNG)